<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                             stop:0 #f8f9fa, 
                             stop:1 #e9ecef);
}

QLabel {
  color: #2c3e50;
  font-size: 15px;
  font-weight: 500;
}

QLineEdit {
  padding: 14px 18px;
  border: 2px solid #dee2e6;
  border-radius: 8px;
  background: #ffffff;
  font-size: 15px;
  color: #212529;
  min-height: 20px;
}

QLineEdit:focus {
  border-color: #3498db;
  background: #ffffff;
  outline: none;
}

QLineEdit:hover {
  border-color: #adb5bd;
}

QPushButton {
  padding: 12px 32px;
  border: none;
  border-radius: 8px;
  font-size: 15px;
  font-weight: 600;
  min-width: 120px;
  min-height: 42px;
}

QPushButton#btnLogin {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                              stop:0 #3498db, 
                              stop:1 #2980b9);
  color: #ffffff;
}

QPushButton#btnLogin:hover {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                              stop:0 #2980b9, 
                              stop:1 #21618c);
}

QPushButton#btnLogin:pressed {
  background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                              stop:0 #21618c, 
                              stop:1 #1a5490);
}

QPushButton#btnCancel {
  background: #6c757d;
  color: #ffffff;
}

QPushButton#btnCancel:hover {
  background: #5a6268;
}

QPushButton#btnCancel:pressed {
  background: #545b62;
}

QCheckBox {
  color: #495057;
  font-size: 14px;
  font-weight: 400;
}

QCheckBox::indicator {
  width: 20px;
  height: 20px;
  border: 2px solid #adb5bd;
  border-radius: 4px;
  background: #ffffff;
}

QCheckBox::indicator:hover {
  border-color: #3498db;
}

QCheckBox::indicator:checked {
  background: #3498db;
  border-color: #3498db;
  image: none;
}

QCheckBox::indicator:checked::after {
  content: "";
  width: 5px;
  height: 10px;
  border: solid white;
  border-width: 0 2px 2px 0;
  transform: rotate(45deg);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>PowerDAQ 登录</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 32px;
font-weight: 700;
color: #212529;
margin-bottom: 15px;
letter-spacing: 1px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 8px;
font-size: 15px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="placeholderText">
      <string>请输入用户名</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_username">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>密码</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 8px;
font-size: 15px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_password">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_checkboxes">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxRemember">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowPassword">
       <property name="text">
        <string>显示密码</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_checkboxes">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_middle">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>登录</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_right">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

